elastic/integrations
# Fix dashboard breakage

## What does this PR do?

See https://github.com/elastic/integrations/issues/505

I'm still trying to test this, as I don't have a great windows environment set up, but if anyone does have an at-hand winlogbeat test environment, please pull this down and test.

## Checklist



## How to test this PR locally

-  Pull down PR
- Build the system package with `elastic-package build`
- Start the stack, with an agent on Windows
- Make sure the Windows dashboards is working.
